examples/suit_update/Makefile.board.dep

```diff
@@ -0,0 +1,7 @@
+ifeq ($(BOARD),native)
+  USEMODULE += suit_storage_ram
+  USEMODULE += gnrc_netdev_default
+else
+  USEMODULE += suit_storage_flashwrite
+endif
+
```

Comment:
Can you squash in this static-test comment as well?

Reply:
Done!